frontend/src/utilities/useWatchQuickStarts.tsx

```diff
@@ -14,7 +14,7 @@ export const useWatchQuickStarts = (): {
   const [quickStarts, setQuickStarts] = React.useState<QuickStart[]>([]);
 
   React.useEffect(() => {
-    let watchHandle: ReturnType<typeof setTimeout>;
+    let watchHandle: ReturnType<typeof setTimeout> | undefined;
```

Comment:
Remove the if condition in the return statement.
```suggestion
    let watchHandle: ReturnType<typeof setTimeout>;
```